vivo-project/Vitro
# 3762 Merging Vitro-Languages into Vitro repository

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3762](https://github.com/vivo-project/VIVO/issues/3762)

# What does this pull request do?
Merge Vitro-languages into the Vitro repository

# What's new?
There is i18n module in the Vitro project. Vitro-languages project is not needed anymore.

# How should this be tested?
1. configure tomcat directory and home directory in installer/example-settings.xml
2. clean home directory and maven repository (org.vivoweb)
3. mvn clean install -o -s installer/example-settings.xml
4. run tomcat
5. check application in the web browser (try to log in and browse a little bit)
6. check structure of <tomcat>/webapps/vivo directory (especially i18n directory, there should be property files and 3 readme files)
7. check structure of <vitro home> directory, especially rdf/i18n/fr_CA/display/firsttime/ (there should be 1 file containing UILabels term, this is just an example, not containing all UI labels, just one)
8. please check display/firsttime directories (point 7) for other languages

# Additional Notes:
* Does this change require documentation to be updated?  **documentation about adding new language should be updated**
* Does this change add any new dependencies? **NO, but the files are reorganized, there are changes in pom files, but no new dependencies**
* Could this change impact execution of existing code? **Hopefully not, but it would be nice to have a test**
* Large pull requests should be avoided. If this PR is large (more than 1,000 lines of codes), please provide short explanation why your contribution can't be decoupled in smaller PRs.  **This is a huge PR, because we need to keep files history, which is moving from the another repository. Please, check only changed files in the last commit.**

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
